{{cookiecutter.repo_name}}/requirements/base.txt

```diff
@@ -1,10 +1,10 @@
-# Django 
-Django==1.9.9
+# Django
+Django==1.9.11
```

Comment:
I'm completely outside the loop (since I'm not actively maintaining the project anymore), but is Django CMS compatible with Django 1.10.x? I think you're not upgrading to the latest version because it's not compatible right?